src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,126 @@
+import { ChangeEvent, FC, useCallback } from 'react';
+import classNames from 'classnames';
+import { URLSearchParams } from 'url';
+import { getSearchWith } from '../../utils/helpers';
+import { SearchLink } from '../SearchLink';
+import { CENTURIES_LIST } from '../../utils/constants';
+
+type Props = {
+  sex: string;
+  query: string;
+  centuries: string[];
+  searchParams: URLSearchParams;
+  setSearchParams: (getSearchWith: string) => void;
+};
+
+export const PeopleFilters: FC<Props> = ({
+  sex,
+  query,
+  centuries,
+  searchParams,
+  setSearchParams,
+}) => {
+  const handleChange = useCallback((e: ChangeEvent<HTMLInputElement>) => {
+    setSearchParams(
+      getSearchWith(searchParams, { query: e.target.value || null }),
+    );
+  }, [searchParams]);
```

Comment:
its better to use full names
```suggestion
  const handleChange = useCallback((event: ChangeEvent<HTMLInputElement>) => {
    setSearchParams(
      getSearchWith(searchParams, { query: event.target.value || null }),
    );
  }, [searchParams]);
```